Config.uk

```diff
@@ -1,13 +1,21 @@
-menuconfig LIBLUA
-	bool "The Lua programming language"
-	default n
-        select LIBMUSL
-        select LIBUKTIME
-        select UKUNISTD
-        select LIBUKSIGNAL
+menu "Library configuration"
+
+config LIBLUA
+    bool "Lua scripting language"
```

Comment:
I understand the using of consistent indents but it needs to use `menuconfig` here to be in line with the other ones